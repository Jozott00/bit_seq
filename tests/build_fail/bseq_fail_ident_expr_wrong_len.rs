use bit_seq::bseq;

fn main() {
    let var = 0;
    bseq!(var:wrong_len);
}