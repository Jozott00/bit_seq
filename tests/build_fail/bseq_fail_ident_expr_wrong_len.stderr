error: expected integer that specifies size of bit sequence
 --> tests/build_fail/bseq_fail_ident_expr_wrong_len.rs:5:15
  |
5 |     bseq!(var:wrong_len);
  |               ^^^^^^^^^
